README.md

```diff
@@ -37,14 +38,22 @@ pod 'XcodeServerSDK'
 
 Create the server config object with the server's URL, username and password.
 ```swift
-let config = XcodeServerConfig(host: "https://127.0.0.1", user: "IRuleBots", password: "superSecr3t")
-let server = XcodeServerFactory.server(config)
+do {
+    let config = XcodeServerConfig(host: "https://127.0.0.1", user: "IRuleBots", password: "superSecr3t")
+} catch {
+    fatalError("Something went wrong!")
```

Comment:
We can also write something more detailed to show what are the possibilities, like:
```swift
do {
    let config = XcodeServerConfig(host: "https://127.0.0.1", user: "IRuleBots", password: "superSecr3t")
} catch ConfigurationErrors.NoHostProvided {
    fatalError("You haven't provided any host")
} catch ConfigurationErrors.InvalidHostProvided(let host){
    fatalError("You've provided invalid host: \(host)")
} catch ConfigurationErrors.InvalidSchemeProvided(let scheme) {
    fatalError("You've provided invalid scheme: \(scheme)")
} catch {
    fatalError("Error, not related to XcodeServerConfig; \(error)")
}
```